app/models/etd.rb

```diff
@@ -0,0 +1,76 @@
+module Bart
```

Comment:
a file named `app/models/etd.rb` should only define a class/module named `Etd` instead of `Bart` as you are doing.